packages/scene-composer/src/components/panels/ComponentEditMenu.tsx

```diff
@@ -0,0 +1,168 @@
+
+  const mapToMenuItem = useCallback(
+    (item: ToolbarItemOptionRaw): ComponentEditMenuItem => {
+      const typeId: ObjectTypes = item.uuid as ObjectTypes;
+      return {
+        ...item,
+        uuid: typeId,
+        subItems: item.subItems?.map(mapToMenuItem),
+        label: formatMessage(labelStrings[typeId]),
+        text: textStrings[typeId] ? formatMessage(textStrings[typeId]) : undefined, // if there's a text string, show text
+      };
+    },
+    [formatMessage],
+  );
+
+  const componentItems = useMemo(() => {
+    switch (currentComponent.type) {
+      case KnownComponentType.DataBinding:
+        return [
+          {
+            uuid: ObjectTypes.AddDataBinding,
+          },
+          {
+            uuid: ObjectTypes.RemoveAllDataBinding,
+          },
+        ];
+
+      case KnownComponentType.DataOverlay: {
+        const items = [
+          {
+            uuid: ObjectTypes.AddDataBinding,
+          },
+        ];
+        if ((currentComponent as IDataOverlayComponentInternal).subType === Component.DataOverlaySubType.OverlayPanel) {
+          items.push({
+            uuid: ObjectTypes.RemoveOverlay,
+          });
+        }
+        return items;
+      }
+      default:
+        return [];
+    }
+  }, [currentComponent]);
+
+  const menuItems = useMemo(
+    () =>
+      [
+        {
+          icon: { name: 'ellipsis' as IconProps.Name },
+          uuid: ObjectTypes.EditComponent,
+        },
+        ...componentItems,
+      ].map(mapToMenuItem),
+    [currentComponent],
+  );
+
+  const handleAddDataBinding = useCallback(() => {
+    switch (currentComponent.type) {
+      case KnownComponentType.DataBinding: {
+        const newComponentPartial = {
+          ...currentComponent,
+          valueDataBindings: [...((currentComponent as IDataBindingComponentInternal).valueDataBindings || []), {}],
+        };
+        updateComponentInternal(nodeRef, newComponentPartial);
+        return;
+      }
+      case KnownComponentType.DataOverlay: {
+        const newComponentPartial = {
+          ...currentComponent,
+          valueDataBindings: [
+            ...((currentComponent as IDataOverlayComponentInternal).valueDataBindings || []),
+            { bindingName: generateUUID() },
+          ],
+        };
+        updateComponentInternal(nodeRef, newComponentPartial);
+        return;
+      }
+      default:
+        throw `Add data binding not implemented for current component type ${currentComponent.type}`;
+    }
+  }, [nodeRef, currentComponent]);
+
+  const handleRemoveAllDataBinding = useCallback(() => {
+    if (currentComponent.type == KnownComponentType.DataBinding) {
+      removeComponent(nodeRef, currentComponent.ref);
+      return;
+    }
+    throw `Remove all data binding not implemented for current component type ${currentComponent.type}`;
+  }, [nodeRef, currentComponent]);
+
+  return menuItems.length > 1 ? (
+    <div style={{ height: '24px' }}>
```

Comment:
Nit: Adding a comment to explain how you got 24px would be helpful.

Reply:
tbh, it's from manual verification to see which value looks better...

Reply:
In general, we should avoid using "px" as a unit in most scenarios. It's a core tenant of responsive design and accessibility. When you have something like a container that's hard coded to 24px high for example, and the user increases their font-size 800% because they have difficulty seeing, it can mess up the layout, or become illegible.

https://betterprogramming.pub/stop-using-the-pixel-unit-in-css-8b8788a1301f -> This article goes into a lot of the reasons you should avoid using this unit of measure.

Also, prefer having a stylesheet to inline styles.

Reply:
I tested to increase my browser's font size, the height of the parent component of this menu component doesn't change with font size change, so it's using "px"? which means this component should use "px" as well to make them work together